database/pgx/pgx.go

```diff
@@ -370,7 +370,7 @@ func (p *Postgres) releaseTableLock() error {
 	}
 
 	query := "DELETE FROM " + pq.QuoteIdentifier(p.config.LockTable) + " WHERE lock_id = $1"
-	if _, err := p.db.Exec(query, aid); err != nil {
+	if _, err := p.conn.ExecContext(context.Background(), query, aid); err != nil {
```

Comment:
I didn't change these functions to accept a context just because I didn't want to deviate too far from the current behavior in which these queries are not cancellable, mostly on the grounds that I'm not super familiar with the ramifications of making them cancellable.

If you or the upstream maintainers would prefer that these functions accept a context rather than use the background context, I'd be happy to make that change!